src/scheduler/job_info.c

```diff
@@ -2767,10 +2770,9 @@ struct preempt_ordering *schd_get_preempt_order(resource_resv *resresv, server_i
 	if (get_job_req_used_time(resresv, &req, &used) != 0)
 		return NULL;
 
-	if (req == -1 || used == -1) {
+	if (req == -1.0 || used == -1.0)
```

Comment:
Just curious, why are these doubles?

Reply:
you mean why req and used are doubles? they are set to 'sch_resource_t' type values inside the function get_job_req_used_time(), which is a double, that's why i made them doubles

Reply:
They don't need to be doubles.  They will never have a floating point value.  I made sch_resource_t a double because I had to choose one type.  It looks strange that you are checking against -1.0 for a walltime type.